src/dart-2.0.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+permalink: /dart-2.0
+title: Dart 2.0 Updates
+description: How Dart 2.0 is different than 1.x, and how you can prepare.
+---
+
+This page has information about changes that are coming in Dart 2.0.
+Some changes may not apply to your target platform.
+
+* [Changes in Dart 2.0](#changes-in-dart-2.0)
+  * [Web](#web)
+  * [Mobile (Flutter)](#mobile-flutter)
+  * [Servers and command line](#servers-and-command-line)
+* [Removed from Dart 2.0](#removed-from-dart-2.0)
+
+---
+
+You can implement these changes now,
+but keep watching this page for updates. We'll let you know what you can
+do now, and what needs to wait for Dart 2.0.
+
+## Changes in Dart 2.0
+
+Dart 1.x is an optionally typed language, but Dart 2.0 requires static types.
```

Comment:
The big change in Dart 2 is that var changes from "dynamic" to "infer me!". This is a HUGE change.

this is valid in Dart 2:

dynamic x = 1;
x = "String";

This is invalid in Dart 2:

var x = 1;
x = "string";

See https://dartpad.dartlang.org/c7d9bb2240ffaba75122a6347a096d29